config/locales/helpers.en.yml

```diff
@@ -0,0 +1,18 @@
+en:
+  helpers:
+    caption: {}
+    hint:
+      teacher_interface_alternative_name_form:
+        has_alternative_name: If your name appears differently on your ID documents or qualifications you need to upload proof of your name change, for example, your marriage or civil partnership certificate.
+        alternative_given_names: Enter your full name apart from your family name
+        alternative_family_name: Enter just your family name
+    label:
+      teacher_interface_alternative_name_form:
+        has_alternative_name_options:
+          true: Yes – I'll upload another document to prove this
```

Comment:
I think this should be a ‘ not a '

Reply:
Good point, I've amended the commit with this change!